### PR TITLE
fix: `activeRequestBias` missing default value (1.0 but not 0.0), and fix typo

### PR DESCRIPTION
**Pull request by jizhuozhi**
The current LAR and LAC load balancing algorithms do not have a test for request distribution, adding tests to avoid untestable changes in the future

### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Use a fixed weight and a fixed number of connections, but use a looser statistical error (delta)

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/upstream/cluster/lb_leastconnection_test.go [hunk 1]
```diff
@@ -53,3 +56,60 @@ func TestLACChooseHost(t *testing.T) {
...
+		compareDistribution(t, expect, actual, delta)
+	}
+
+	t.Run("no bias", func(t *testing.T) {
+		verify(t, nil, 0.15)
+	})
+	t.Run("bias 0.9", func(t *testing.T) {
+		verify(t, &clusterInfo{
+			lbConfig: &v2.LbConfig{
+				ActiveRequestBias: 0.5,
```
Comment (spacewander) [inline pkg/upstream/cluster/lb_leastconnection_test.go:104]: Why use 0.5 when the test name is "bias 0.9"?

Comment (jizhuozhi) [reply pkg/upstream/cluster/lb_leastconnection_test.go:104]: my mistake, renamed

**Comment by jizhuozhi** [comment]
Sorry, I found my mistake. When multiple branches are switched, some commits are not pushed. `activeRequestBias` has no default value, resulting in break change, the original semantic `activeRequestBias` is 1, but now it is 0.

I discovered the problem when these commits were rebased into this PR. I'm so sorry I almost caused a serious production accident.

cc @doujiang24 , @taoyuanyuan , @3062 , @spacewander

**Review by doujiang24 — DISMISSED** [review]

**Comment by taoyuanyuan** [comment]
pls resolve conflict.

**Review by taoyuanyuan — APPROVED** [review]

**Review by doujiang24 — APPROVED** [review]

**Pull request merged**